Marlin/src/pins/stm32f1/pins_BTT_SKR_MINI_E3_common.h

```diff
@@ -124,18 +124,35 @@
  *                 ------                                  ------
  *                  EXP1                                    EXP1
  */
-#ifndef EXP1_02_PIN
+#ifndef EXP1_08_PIN
+  #define EXP1_01_PIN                       PB5
   #define EXP1_02_PIN                       PB6
+  #define EXP1_03_PIN                       PA9
+  #define EXP1_04_PIN                       -1    // RESET
+  #define EXP1_05_PIN                       PA10
+  #define EXP1_06_PIN                       PB9
+  #define EXP1_07_PIN                       PB8
   #define EXP1_08_PIN                       PB7
 #endif
-#define EXP1_01_PIN                         PB5
-#define EXP1_03_PIN                         PA9
-#define EXP1_04_PIN                         -1    // RESET
-#define EXP1_05_PIN                         PA10
-#define EXP1_06_PIN                         PB9
-#define EXP1_07_PIN                         PB8
-
-#if HAS_DWIN_E3V2 || IS_DWIN_MARLINUI
+
+/*   -----
+ *   | 1 | RST
+ *   | 2 | PA3 RX2
+ *   | 3 | PA2 TX2
+ *   | 4 | GND
+ *   | 5 | 5V
+ *   -----
+ *    TFT
+ */
+
+#define TFT_02                              PA3
+#define TFT_03                              PA2
```

Comment:
(Also applies to E3 DIP board)

I don't really like these names `TFT_02` and `TFT_03`. I see that these are just documenting pin numbers in the connector on the board, but the `TFT_` prefix is used by many TFT-related configuration options as well.

Perhaps this should be more clearly scoped to the board, to avoid confusing readers into thinking these are consumed directly by TFT code elsewhere.

Maybe `BOARD_TFT_02`?

Reply:
On another note, this is a serial port so I assume it's used as such, so we may just end up using the define `LCD_SERIAL_PORT` for displays like this and define a dummy "adapter" that simply indicates the selected LCD needs a serial port.